package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "frisby",
-  "version": "0.8.3",
-  "description": "Frisby.js: REST API Endpoint Testing built on Jasmine",
+  "name": "frisby-hack",
```

Comment:
Yeah... there is no way I can ever pull this into the project with a modified package.json like this.